docs: add Related Benchmarks section and cross-link AssetOpsBench (#881)
itbench-hub/ITBench: README.md

# ITBench

**[Paper](./it_bench_arxiv.pdf) | [Leaderboard](#leaderboard) | [Scenarios](#scenarios) | [Agents](#agents) | [How to Cite](#how-to-cite) | [Contributors](./CONTRIBUTORS.md) | [Contacts](#contacts)**
**[Paper](./it_bench_arxiv.pdf) | [Leaderboard](#leaderboard) | [Scenarios](#scenarios) | [Agents](#agents) | [Related Benchmarks](#related-benchmarks) | [How to Cite](#how-to-cite) | [Contributors](./CONTRIBUTORS.md) | [Contacts](#contacts)**

---

## 📢 Announcements

### Latest Updates
- **[May 27, 2026]** Artificial Analysis and IBM Research launched **ITBench-AA**, the first in a new series of benchmarks evaluating frontier models on agentic enterprise IT tasks—starting with 59 SRE tasks where all evaluated models score below 50%, with FinOps and CISO tasks to follow. [View the evaluation](https://artificialanalysis.ai/evaluations/itbench-aa).

> - **[January 21, 2026]** IBM Research has published the **Enterprise Agents and Benchmarks** collection on Hugging Face, featuring ITBench alongside other enterprise AI agent ecosystems and benchmarks. [View the collection](https://huggingface.co/collections/ibm-research/enterprise-agents-and-benchmarks).

Reviewer: Please re-add these two updates.

> - **[January 21, 2026]** IBM Research has published the **Enterprise Agents and Benchmarks** collection on Hugging Face, featuring ITBench alongside other enterprise AI agent ecosystems and benchmarks — including the new sibling project **[AssetOpsBench](https://github.com/IBM/AssetOpsBench)** for industrial asset operations & maintenance. [View the collection](https://huggingface.co/collections/ibm-research/enterprise-agents-and-benchmarks) · See the new [Related Benchmarks](#related-benchmarks) section below.

Reviewer: ```suggestion
- **[January 21, 2026]** IBM Research has published the **Enterprise Agents and Benchmarks** collection on Hugging Face, featuring ITBench alongside other enterprise AI agent ecosystems and benchmarks — including the new sibling project **[AssetOpsBench](https://github.com/IBM/AssetOpsBench)** for industrial asset operations & maintenance. [View the collection](https://huggingface.co/collections/ibm-research/enterprise-agents-and-benchmarks)
```

---

## Related Benchmarks

> ITBench is part of IBM Research's **[Enterprise Agents and Benchmarks](https://huggingface.co/collections/ibm-research/enterprise-agents-and-benchmarks)** collection on Hugging Face — a family of open benchmarks for evaluating AI agents on enterprise tasks. If you are working across this ecosystem, you may also be interested in:

Reviewer: ```suggestion
ITBench is part of IBM Research's **[Enterprise Agents and Benchmarks](https://huggingface.co/collections/ibm-research/enterprise-agents-and-benchmarks)** collection on Hugging Face, a family of open benchmarks for evaluating AI agents on enterprise tasks. This collection aims to provide reproducible scenarios and evaluation pipelines for domain-specific AI agents on real-world enterprise workflows. For information on these and other related projects, please see the table below:
```

| Benchmark | Focus | Repository |
|-----------|-------|------------|
| **ITBench** | IT automation: SRE, CISO (compliance & security), FinOps | [itbench-hub/ITBench](https://github.com/itbench-hub/ITBench) |
| **AssetOpsBench** | Industrial asset operations & maintenance (Industry 4.0); LLMs + Time Series Foundation Models over sensor data, FMEA, and work orders | [IBM/AssetOpsBench](https://github.com/IBM/AssetOpsBench) |

Both projects share a common goal: providing reproducible scenarios and evaluation pipelines for domain-specific AI agents on real-world enterprise workflows.

> ---

Reviewer: ```suggestion
```
